src/vs/workbench/contrib/terminalContrib/suggest/browser/lspTerminalModelContentProvider.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { Disposable } from '../../../../../base/common/lifecycle.js';
+import { IModelService } from '../../../../../editor/common/services/model.js';
+import { ILanguageService } from '../../../../../editor/common/languages/language.js';
+import { ITextModelContentProvider, ITextModelService } from '../../../../../editor/common/services/resolverService.js';
+import { URI } from '../../../../../base/common/uri.js';
+import { ITextModel } from '../../../../../editor/common/model.js';
+import { Schemas } from '../../../../../base/common/network.js';
+
+// TODO: Where do I instantiate this? My guess is `terminal.suggest.contribution.ts`
```

Comment:
Debating where I should instantiate this TextModelContentProvider.
Most likely in src/vs/workbench/contrib/terminalContrib/suggest/browser/terminal.suggest.contribution.ts, since I already load `lspCompletionAddon`?